apps/test/unit/templates/studioHomepages/StudentHomepageTest.js

```diff
@@ -57,6 +60,13 @@ describe('StudentHomepage', () => {
     });
   });
 
+  it('does not log an Amplitude event for teacher signing-in', () => {
```

Comment:
nit: should this test name be "does not log an Amplitude event for **student** signing-in"?